src/pages/home/report/ReportActionItemCreated.js

```diff
@@ -38,28 +41,46 @@ const defaultProps = {
 const ReportActionItemCreated = (props) => {
     const isPolicyExpenseChat = ReportUtils.isPolicyExpenseChat(props.report);
     const icons = ReportUtils.getIcons(props.report, props.personalDetails, props.policies);
+
+    // The image handling depends on how large the chat window is
+    const backgroundImageHeight = Dimensions.get('window').height < Dimensions.get('window').width
+        ? 450
+        : 300;
```

Comment:
What's the reasoning behind 450 / 300?

Reply:
The 300 is what was needed to match the iOS mocks. The 450 is the smallest possible height to have it not cut off by the container top edge on desktop at bigger screen sizes. I wish I had better answers for you, haha.

Reply:
in that case, maybe we should add these to the `variables.js` file and reference from there? Then you could use a more descriptive name like imagePortraitWidth / imageLandscapeWidth (just an example)

Reply:
Follow up: Is it possible to use % here so you wouldn't have to reset the height and width later?

Reply:
I think I could swap to percentage here but it would still have to be two different values... I can look into it though!